cl_dll/hud_settings.cpp

```diff
@@ -49,36 +49,40 @@ int CHudSettings::Draw(float time)
 
 	char str[32];
 
-	sprintf(str, "AG version %s", ag_version);
-	gEngfuncs.pfnDrawString(x, (y += gHUD.m_scrinfo.iCharHeight), str, r, g, b);
-
-	gEngfuncs.pfnDrawString(x, (y += gHUD.m_scrinfo.iCharHeight), gamemode, r, g, b);
-
-	sprintf(str, "Time limit: %hhd", time_limit);
-	gEngfuncs.pfnDrawString(x, (y += gHUD.m_scrinfo.iCharHeight / 2 * 3), str, r, g, b);
-
-	sprintf(str, "Frag limit: %hhd", frag_limit);
-	gEngfuncs.pfnDrawString(x, (y += gHUD.m_scrinfo.iCharHeight), str, r, g, b);
+	extern cvar_t* hud_watermark;
+	if (hud_watermark->value == 1)
```

Comment:
But this is settings and not watermark?

Reply:
I thought it would make sense for `hud_watermark` to control both components as apart from their names in the code they are essentially watermarks that appear on either side of the screen. I should have clarified this in the description. Should this be removed for it to only be controlled by `hud_settings`?

Reply:
Maybe it would be best to ask ppl in the AG discord what they think?

Reply:
![Screenshot_20241030_113550.jpg](https://github.com/user-attachments/assets/30af6554-cbdb-48c1-b7f2-1e44edb07ef8)

It would appear that the discord has decided. Still though, up to you if you want to implement this change :)